docs/api-guide/viewsets.md

```diff
@@ -137,7 +137,7 @@ For example:
         def recent_users(self, request):
             recent_users = User.objects.all().order('-last_login')
             page = self.paginate_queryset(recent_users)
-            serializer = self.get_pagination_serializer(page)
+            serializer = self.get_paginated_response(page)
```

Comment:
This change isn't correct. This method will return the response object itself, not a serializer.